rule/rule.go

```diff
@@ -816,6 +816,27 @@ func (r *Rule) sync() {
 	r.updated = false
 }
 
+// AttrEqual decides if the rule and the other rule has the same value on the attribute
+func (r *Rule) AttrEqual(other *Rule, key string) bool {
+	value := r.AttrString(key)
+	if len(value) > 0 && value == other.AttrString(key) {
+		return true
+	}
+	myValues := r.AttrStrings(key)
```

Comment:
Rename `myValues` to `rValues` (or `xValues` and `yValues` when this is moved to `merger`).